packages/docs/src/pages/guides/mdx-layout-components.mdx

```diff
@@ -27,6 +27,7 @@ export default props => (
 To add styles to this component, add a `theme` prop to the `ThemeProvider` and an `sx` prop to the `div`.
 
 ```jsx
+// nile name: MyLayout.js
```

Comment:
Thanks for the suggestion, but I think we should solve this for all code blocks using a [code block meta string](https://mdxjs.com/guides/live-code#code-block-meta-string)

```suggestion
```jsx filename=Layout.js
```